pkg/Directory.Build.props

```diff
@@ -33,6 +33,12 @@
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\lib\**\*" Pack="true" PackagePath="lib" />
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\analyzers\**\*" Pack="true" PackagePath="analyzers" />
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\runtimes\**\*" Pack="true" PackagePath="runtimes" />
+    <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\tools\**\*" Pack="true" PackagePath="tools" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IncludeMLNetNotices)' != 'false'">
```

Comment:
This appears to be duplicated now with the section below it. I believe it was a bad merge.